### ISSUE TITLE
Train with negative sample (No table in image)

### ISSUE THREAD

**Issue by namirinz** [microsoft/table-transformer#113]
Hi I would like to know how to train table detection or does it support if there are not any table in the image and XML of that image wouldn't have object.
I tried to train but it give me an error which declare that there are no label (object) in XML file.

![Screenshot_20230427_111507](https://user-images.githubusercontent.com/35620424/234757921-3dbe83e4-8165-4469-9b82-140a549a5478.png)

![Screenshot_20230427_111604](https://user-images.githubusercontent.com/35620424/234758035-eb96ed72-02ab-487c-bf8d-5247e63cabff.png)

**Comment by bsmock** [comment]
Yes, this is possible when training DETR, but we did not write the data loader to handle empty training samples.

I believe it would be just a few lines of code to make the change. If you make the change yourself, feel free to submit a pull request!

**Comment by mspondon** [comment]
Hey Guys, Any update on this ?
My concern is the above detection model detect a non tabular region as a table.
I want to finetune the model with negative sample.
